### ISSUE TITLE
Should property setters be violated?

### ISSUE THREAD

**Issue by eremeevfd** [pwoolvett/flake8_boolean_trap#5]
Should I just leave noqa or maybe this behaviour should be changed?
```python
class MyClass:
    @property
    def some_attribute(self) -> bool:
        return self._attribute

    @some_attribute.setter
    def some_attribute(self, value: bool) -> None:
        self._atrribute = value
```
It's impossible to create setter with kwargs

**Comment by pwoolvett** [comment]
You are right. Property setters should be fine.

Could you take a look at #6 and give some feedback?

**Comment by pwoolvett** [comment]
solved by #6
